### PR TITLE
Add project initiatives

### LINKED ISSUE #4: Add current projects to main README
as titled - this was an action item from previous meetings

### PR DESCRIPTION
**Pull request by Ethan-Arrowood**
closes #4

### CODE REVIEW

**Review by rmarkins-godaddy — APPROVED** [review]
LGTM

**Review by rxmarbles — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,2 +1,16 @@
...
+
 The goal of the Package Metadata Interoperability Collab Space is to improve how JavaScript developers define their packages across the ecosystem. The group is currently working to better understand package.json and how developers use it to define their projects.
+
+## Current Initiatives
+
+1. [package.json research](https://github.com/openjs-foundation/package-json-research)
+   1. This initiative's goal is to research existing usage of `package.json` throughout the JavaScript ecosystem
+   2. With this research, the group can develop tooling, documentation, specifications, and much more
+   3. The linked repository contains all information for how to get involved and contribute
+2. [`@pkgjs/create-pkg` tooling](https://github.com/pkgjs/create-pkg)
```
Comment (AugustinMauroy) [inline README.md:11]: GitHub link is 404. is this a private repo?

Comment (ljharb) [reply README.md:11]: it's public, and https://github.com/pkgjs/create-pkg loads for me.

**Comment by AugustinMauroy** [comment]
@ljharb after a reload it is aparus surely a problem of connection today I have full of problems 😅

File: README.md [hunk 2]
```diff
@@ -1,2 +1,16 @@
...
+   1. An initiative originally created by Node.js collaborators for Node.js based package tooling, the efforts here are now being expanded to align with this group's wider scope.
+   2. The tooling being developed will be package manager and runtime agnostic and help developers across the entire JavaScript ecosystem
```
Comment (GeoffreyBooth) [inline README.md:13]:
```suggestion
   1. This initiative's goal is to research existing usage of `package.json` throughout the JavaScript ecosystem.
   2. With this research, the group can develop tooling, documentation, specifications, and much more.
   3. The linked repository contains all information for how to get involved and contribute.
2. [`@pkgjs/create-pkg` tooling](https://github.com/pkgjs/create-pkg)
   1. An initiative originally created by Node.js collaborators for Node.js based package tooling, the efforts here are now being expanded to align with this group's wider scope.
   2. The tooling being developed will be package manager and runtime agnostic and help developers across the entire JavaScript ecosystem.
```

**Review by GeoffreyBooth — APPROVED** [review]

**Pull request merged**